mmcv/utils/config.py

```diff
@@ -176,7 +196,9 @@ def _substitute_base_vars(cfg, base_var_dict, base_cfg):
         return cfg
 
     @staticmethod
-    def _file2dict(filename, use_predefined_variables=True):
+    def _file2dict(filename,
```

Comment:
Should we add argument `use_environment_variables` for `fromfile` function, as it call `_file2dict`